SPLASH/src/game/Game.cpp

```diff
@@ -13,7 +13,7 @@ Game::Game(HINSTANCE hInstance)
 	registerStates();
 
 	// Set starting state
-	m_stateStack.pushState(States::MainMenu);
+	m_stateStack.pushState(States::Game);
```

Comment:
Vill vi att programmet startar i Game och inte i MainMenu?

Reply:
Forgot about that, will fix!